astropy/cosmology/realizations.py

```diff
@@ -72,80 +73,23 @@ class default_cosmology(ScienceState):
 
         >>> default_cosmology.get()
         FlatLambdaCDM(name="Planck18", H0=67.66 km / (Mpc s), Om0=0.30966, ...
-
-    To get a specific cosmology:
-
-        >>> default_cosmology.get("Planck13")
-        FlatLambdaCDM(name="Planck13", H0=67.77 km / (Mpc s), Om0=0.30712, ...
     """
 
     _default_value = "Planck18"
     _value = "Planck18"
 
-    @classmethod
-    def get(cls, key=None):
-        """Get the science state value of ``key``.
-
-        Parameters
-        ----------
-        key : str or None
-            The built-in |Cosmology| realization to retrieve.
-            If None (default) get the current value.
-
-        Returns
-        -------
-        `astropy.cosmology.Cosmology` or None
-            `None` only if ``key`` is "no_default"
-
-        Raises
-        ------
-        TypeError
-            If ``key`` is not a str, |Cosmology|, or None.
-        ValueError
-            If ``key`` is a str, but not for a built-in Cosmology
-
-        Examples
-        --------
-        To get the default cosmology:
-
-        >>> default_cosmology.get()
-        FlatLambdaCDM(name="Planck18", H0=67.66 km / (Mpc s), Om0=0.30966, ...
-
-        To get a specific cosmology:
-
-        >>> default_cosmology.get("Planck13")
-        FlatLambdaCDM(name="Planck13", H0=67.77 km / (Mpc s), Om0=0.30712, ...
-        """
-        if key is None:
-            key = cls._value
-
-        if isinstance(key, str):
-            # special-case one string
-            if key == "no_default":
-                return None
-            # all other options should be built-in realizations
-            try:
-                value = getattr(sys.modules[__name__], key)
-            except AttributeError:
-                raise ValueError(f"Unknown cosmology {key!r}. "
-                                 f"Valid cosmologies:\n{available}")
-        elif isinstance(key, Cosmology):
-            value = key
-        else:
-            raise TypeError("'key' must be must be None, a string, "
-                            f"or Cosmology instance, not {type(key)}.")
-
-        # validate value to `Cosmology`, if not already
-        return cls.validate(value)
-
     @deprecated("5.0", alternative="get")
```

Comment:
Can still use `.get()`, right? Just uses one from the superclass.

Reply:
Yes. Back to basics. ``.get()`` on the superclass just calls validate.